testWorkspace/tsNodeApp/.vscode/launch.json

```diff
@@ -15,7 +15,8 @@
             "program": "${workspaceFolder}\\app.ts",
             "outFiles": [
                 "${workspaceFolder}/**/*.js"
-            ]
+            ],
+            "customDescriptionGenerator": "const lal = 4; lal + global.customDebuggerDescription(this, defaultValue)",
```

Comment:
Probably undo the changes to this demo -- or move it to a different demo for your work

Reply:
Moved to it's own demo